web/src/pages/Route/components/DebugViews/DebugDrawView.tsx

```diff
@@ -229,8 +233,23 @@ const DebugDrawView: React.FC<RouteModule.DebugDrawProps> = (props) => {
     }, bodyFormData)
       .then((req) => {
         setLoading(false);
-        setResponseBody(JSON.stringify(req.data.data, null, 2));
-        setResponseHeader(JSON.stringify(req.data.header, null, 2));
+        const resp: RouteModule.debugResponse= req.data;
+        if (typeof (resp.data) !== 'string') {
```

Comment:
> It would be better to use `resp.header['Content-Type']` as judging condition, just like Line244 does

In the backend implementation, if it can be parsed as json then return object type otherwise return string type, there is no check on content-type.

Reply:
actually when it can be parsed as json, the response Content-type will be `application/json`, so you can try to reuse line244 to format response.data, and put `setResponse(resp)` after line252 to simplify the logic.

Reply:
actually when it can be parsed as json, the response Content-type will be `application/json`, so you can try to reuse line244 to format response.data, and put `setResponse(resp)` after line252 to simplify the logic.

Reply:
For example, if the Content-Type `is text/plain` but the returned content is `{}`, the backend will parse it as json type. I don't know why it was designed this way in the first place, but if I change the backend code, this PR will not be backward compatible.

Reply:
got it, thanks for your interpretation, just cc @Jaycean to take a look, this part may need to be optimized in a new pr.